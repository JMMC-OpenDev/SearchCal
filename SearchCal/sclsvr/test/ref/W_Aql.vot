version https://git-lfs.github.com/spec/v1
oid sha256:6eb7def5dac9c22f5fd785ad03afb550262186904e4debcfeeacd13d7ff9b81d
size 120213
