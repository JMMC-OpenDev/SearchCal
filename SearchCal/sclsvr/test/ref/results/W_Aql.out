version https://git-lfs.github.com/spec/v1
oid sha256:3f290c6bd3cf1da41dc755ae09608067974598188029a6c86b8d45103d3ea915
size 73400
